build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+buildscript {
+    val kotlin_version by extra("1.3.71")
+    repositories {
+        jcenter()
+        google()
+        maven { url = uri("http://oss.sonatype.org/content/repositories/snapshots/") }
+    }
+    dependencies {
+        classpath("com.vanniktech:gradle-maven-publish-plugin:0.9.0")
```

Comment:
For better version management you could have `buildSrc` approach, let me know and I can do it for you in a separate PR